lib/msal-common/src/unifiedCache/serialize/Deserializer.ts

```diff
@@ -9,26 +9,13 @@ import { AccessTokenEntity } from "../entities/AccessTokenEntity";
 import { RefreshTokenEntity } from "../entities/RefreshTokenEntity";
 import { AppMetadataEntity } from "../entities/AppMetadataEntity";
 import { CacheHelper } from "../utils/CacheHelper";
-import {
-    AccountCacheMaps,
-    IdTokenCacheMaps,
-    AccessTokenCacheMaps,
-    RefreshTokenCacheMaps,
-    AppMetadataCacheMaps,
-} from "../serialize/JsonKeys";
-import {
-    AccountCache,
-    IdTokenCache,
-    AccessTokenCache,
-    RefreshTokenCache,
-    AppMetadataCache,
-    InMemoryCache,
-    JsonCache,
-} from "../utils/CacheTypes";
+import { AccountCacheMaps, IdTokenCacheMaps, AccessTokenCacheMaps, RefreshTokenCacheMaps, AppMetadataCacheMaps } from "../serialize/JsonKeys";
```

Comment:
Let's agree on a format for imports. I think in most other places we are importing each class in a different line.

Reply:
yep. Will be in a different line.